plugins/auth-backend/README.md

```diff
@@ -165,3 +165,12 @@ To try out SAML, you can use the mock identity provider:
 ## Links
 
 - [The Backstage homepage](https://backstage.io)
+
+## Configuring Token Expiration in App Config
+
+The expiration feature is not enabled unless you set this in your config file:
```

Comment:
```suggestion
If you need to change Backstage token expiration from the default value of one hour, set the following in your config file:
```
Sorry, this wording was not right. Expiration is definitely always enabled for security reasons!